sci-mathematics/gimps/gimps-29.4b5.recipe

```diff
@@ -0,0 +1,70 @@
+SUMMARY="Finds new Mersenne primes"
+DESCRIPTION="GIMPS stands for the \"Great Internet Mersenne Prime Search\". It's a \
+collaborative project of volunteers who use freely available software to \
+search for Mersenne prime numbers. To perform its testing, the project \
+relies primarily on Lucas-Lehmer primality test, an algorithm that is both \
+specialized to testing Mersenne primes and particularly efficient on binary \
+computer architectures. They also have a trial division phase, used to rapidly \
+eliminate Mersenne numbers with small factors which make up a large proportion \
+of candidates. Pollard's p - 1 algorithm is also used to search for larger \
+factors."
+HOMEPAGE="https://www.mersenne.org/"
+COPYRIGHT="1996-2017 Mersenne Research, Inc."
+LICENSE="GIMPS"
+REVISION="1"
+SOURCE_URI="http://www.mersenne.org/ftp_root/gimps/p95v${portVersion//.}.source.zip"
+SOURCE_DIR=""
+CHECKSUM_SHA256="21cdd387e73c3386389386b480d0f7f97073dcd70a4dcf9031112f575594443e"
+
+ARCHITECTURES="x86_gcc2 x86"
```

Comment:
Did you mean `ARCHITECTURES="x86_gcc2 x86 x86_64"` ?
(Sorry if this was on purpose.)

Reply:
I cannot get it to build on x86_64 (linking problems with the factor object file), will mark it as broken.

Reply:
If there is floating point exception it is broken on x86 as well.